playbook/app/pb_kits/playbook/pb_title/docs/_title_light.md

```diff
@@ -0,0 +1,6 @@
+##### Prop
+This kit can accept these colors with the color prop.
+
+* `default` `light` `lighter` `success` `error` `link`
+
```

Comment:
It's worth mentioning that default color is hard-coded in the kit and can be replaced with those colors.

Reply:
good call I just updated it